frontend/src/modules/history/components/History.js

```diff
@@ -73,7 +73,7 @@ export default class History extends React.Component<Props> {
                         deleteTranslation={ deleteTranslation }
                         updateEditorTranslation={ updateEditorTranslation }
                         updateTranslationStatus={ updateTranslationStatus }
-                        key={ key }
+                        index={ index }
```

Comment:
You'll need to keep the `key` part around, React requires that you pass a unique `key` attribute to elements of a list. You can for example use `key={ key } index={ key }` to also pass that down to the component.